portal/templates/portal/base.html

```diff
@@ -277,7 +277,8 @@
     }
 
     $(document).ready(function() {
-        animateScroll();
+        var bannerHeight = $('.sticky-subnav').height();
+        animateScroll(bannerHeight);
```

Comment:
does this work if we are on the same page and we want to scroll to a section?

Reply:
yes?

Reply:
not sure what you mean here - but when we're in KS1 and we click on KS3 it goes there without issues and is displayed as expected

Reply:
it also works in the "terms and conditions" section. (We still have the pre-existing issue when we have the two-banner page though)

Reply:
Ah I think it's fine, I think it's just my rusty knowledge of jQuery, I'll resolve the discussion